webant/templates/recents.html

```diff
@@ -0,0 +1,90 @@
+{% extends "bootstrap/base.html" %}
+{% import "bootstrap/fixes.html" as fixes %}
+
+{% block title %}
+Libreant | {%trans%}Recents{%endtrans%}
+{% endblock %}
+
+{% block styles %}
+ {{super()}}
+<link href="{{ url_for('static', filename='item_list.css') }}" rel="stylesheet">
+<style>
+header{
+    margin-top:10px;
+    margin-bottom: 20px;
+}
+header h1{
+    margin-top:0px;
+    margin-bottom: 0px;
+    font-size: 2.3em;
+    
+}
+</style>
+{% endblock %}
+
+{% block navbar %}
+{% import 'navbar.html' as navbar %}
+{{ navbar.navbar(search=True) }}
+{% endblock %}
+
+{% block content %}
+<div class="container">
+    <header>
+        <h1>{%trans%}Recently added items{%endtrans%}</h1>
+    </header>
+
+    {% if not items %}
+
+    {%trans%}Sorry, no items added since now{%endtrans%}
+
+    {% else %}
+
+    <div id="item-list">
+        {% for b in items %}
+        <div class="item-div row" onclick="window.open('{{url_for('view_book', bookid=b['_id'])}}')">
```

Comment:
This was already clear for me. 
I asked you to provide the link from which you learned that ```onclick``` attribute is deprecated.